UPGRADE-5.3.md

```diff
@@ -43,6 +43,8 @@ FrameworkBundle
  * Deprecate the `session` service and the `SessionInterface` alias, use the `\Symfony\Component\HttpFoundation\Request::getSession()` or the new `\Symfony\Component\HttpFoundation\RequestStack::getSession()` methods instead
  * Deprecate the `KernelTestCase::$container` property, use `KernelTestCase::getContainer()` instead
  * Rename the container parameter `profiler_listener.only_master_requests` to `profiler_listener.only_main_requests`
+ * Deprecate the public `workflow.abstract` and `state_machine.abstract` abstract definitions to private
```

Comment:
This line is better than nothing I think :sweat_smile:

Reply:
I would suggest removing that line, the next one is enough